### PR TITLE
Make snapshot unittest easily updatable

### PR DESCRIPTION
**Pull request by ossy-szeged**
JerryScript-DCO-1.0-Signed-off-by: Csaba Osztrogonác oszi@inf.u-szeged.hu

### CODE REVIEW

File: tests/unit-core/test-snapshot.c [hunk 1]
```diff
@@ -234,8 +234,27 @@ main (void)
...
+
+    if (sizeof (expected_data) != snapshot_size || memcmp (expected_data, snapshot_buffer, sizeof (expected_data)))
+    {
+      printf ("Snapshot data is changed, please update tests/unit-core/test-snapshot.c.\n");
+      printf ("-------------------------------------------------------------------------\n");
+      printf ("    const uint8_t expected_data[] =\n");
+      printf ("    {");
+      for (unsigned int i = 0; i < snapshot_size; i++)
+      {
+        if ((i%8) == 0)
```
Comment (rerobika) [inline tests/unit-core/test-snapshot.c]: spaces around `%`

File: tests/unit-core/test-snapshot.c [hunk 2]
```diff
@@ -234,8 +234,27 @@ main (void)
       0x46, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
       0x14, 0x00, 0x73, 0x74, 0x72, 0x69, 0x6E, 0x67,
       0x20, 0x66, 0x72, 0x6F, 0x6D, 0x20, 0x73, 0x6E,
-      0x61, 0x70, 0x73, 0x68, 0x6F, 0x74
+      0x61, 0x70, 0x73, 0x68, 0x6F, 0x74,
     };
+
+    if (sizeof (expected_data) != snapshot_size || memcmp (expected_data, snapshot_buffer, sizeof (expected_data)))
+    {
+      printf ("Snapshot data is changed, please update tests/unit-core/test-snapshot.c.\n");
```
Comment (rerobika) [inline tests/unit-core/test-snapshot.c]: `is` -> `has`

Comment (zherczeg) [reply tests/unit-core/test-snapshot.c]: has been

Comment (DanielBallaSZTE) [reply tests/unit-core/test-snapshot.c]: `has been` puts more stress on the fact that _someone_ is responsible for the data change, while `has` just indicates that the data differentiates from what the test would expect.

File: tests/unit-core/test-snapshot.c (hunk 1 above)
Comment (ossy-szeged) [reply tests/unit-core/test-snapshot.c]: Good point, fixed.

File: tests/unit-core/test-snapshot.c (hunk 2 above)
Comment (ossy-szeged) [reply tests/unit-core/test-snapshot.c]: I vote for "has been", updated.

**Pull request merged**